resources/assets/sass/pages/home/_informationsection.scss

```
@@ -1,7 +1,7 @@
 // informationsection
 
 .information-section{
-    width: 100%;
+    width: 1150px;
```

You must use `max-width` instead of `width` because the content cant responsive on > 800px, < 1050px
```suggestion
    max-width: 1150px;
```